arch/arm/soc/arm/mps2/arm_mpu_regions.c

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2017 Linaro Limited.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <arch/arm/cortex_m/mpu/arm_mpu.h>
+
+#define FLASH_BASE_ADDRESS (0x00000000)
```

Comment:
Why aren't these coming from DT?

Reply:
we do not have a DTS file for mps2_an385. So currently this will be kept here until we have one.

Reply:
> we do not have a DTS file for mps2_an385. So currently this will be kept here until we have one.

boards/arm/mps2_an385/mps2_an385.dts ???